manifests/namespace-rbac.yaml

```diff
@@ -96,3 +96,22 @@ roleRef:
   kind: ClusterRole
   name: storage-version-migration-initializer
   apiGroup: rbac.authorization.k8s.io
+---
```

Comment:
> The `storage-version-migration-migrator` gives the migrator the ability to get, list, update all resources, is that not enough?

@sanchezl fix

Reply:
From https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kube-storage-version-migrator/47/pull-kube-storage-version-migrator-manually-launched-e2e/1222290262227685376/build-log.txt

```
I0128 23:00:56.303078       1 round_trippers.go:443] PUT https://10.0.0.1:443/apis/rbac.authorization.k8s.io/v1/clusterroles/edit 403 Forbidden in 39 milliseconds
I0128 23:00:56.303160       1 round_trippers.go:449] Response Headers:
I0128 23:00:56.303170       1 round_trippers.go:452]     Audit-Id: aed0307a-49ed-4e0b-9514-92ed370669b7
I0128 23:00:56.303176       1 round_trippers.go:452]     Content-Type: application/json
I0128 23:00:56.303182       1 round_trippers.go:452]     Date: Tue, 28 Jan 2020 23:00:56 GMT
I0128 23:00:56.303455       1 request.go:1017] Response Body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"clusterroles.rbac.authorization.k8s.io \"edit\" is forbidden: user \"system:serviceaccount:kube-system:default\" (groups=[\"system:serviceaccounts\" \"system:serviceaccounts:kube-system\" \"system:authenticated\"]) is attempting to grant RBAC permissions not currently held:
```

Reply:
Can you remove the `storage-version-migration-migrator` role and fix the `storage-version-migration-migrator` role binding?

I agree that the migrator does need the cluster-admin power. Otherwise it can hit the pasted error when migrating roles/clusterroles due to priviledge escalation.